test-requirements.txt

```diff
@@ -0,0 +1,7 @@
+charms.reactive
+flake8
+black
+stestr
+git+https://github.com/juju/charm-helpers
```

Comment:
The charm-helpers project is unfortunately a notorious offender in terms of clean execution. I see there is code in the next patch to mock it out, is this needed?

Reply:
The `mock_charmhelpers` function lives in the `charms_openstack` project. If it requires charmhelpers to run I'd consider it a bug.

Now that you mention it I remember someone breaking it in this way, so i guess we may have to live with it unless we see a way to fix it.